mesos-swarm-marathon/agent-0.json

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json# ",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newStorageAccountNamePrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name Prefix for the Storage Account where the Virtual Machine's disks will be placed.  StorageAccounts may contain at most variables('vmsPerStorageAccount')"
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "agentVMSize": {
+      "type": "string",
+      "metadata": {
+        "description": "The VM role size of the agent node(s)"
+      }
+    },
+    "agentCount": {
+      "type": "int",
+      "metadata": {
+        "description": "The count of agent nodes"
+      }
+    },
+    "masterCount": {
+      "type": "int",
+      "metadata": {
+        "description": "The count of master nodes"
+      }
+    },
+    "subnetPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "The network subnet"
+      }
+    },
+    "subnetRef": {
```

Comment:
This can become a variable to reduce the number of parameters a user has to enter